### PR TITLE
Allows setting a fallback key for the PostgresPgp decryptor

### PR DESCRIPTION
**Pull request by fabiokr**
This can be used when needing to rotate keys for example. By configuring
a fallback key, you can setup a new key that is used for new
encryptions. For decryptions, it will try to decrypt using the key
first. If that fails, it will also try using the fallback key.

### CODE REVIEW

File: lib/crypt_keeper/provider/postgres_pgp.rb [hunk 1]
```diff
@@ -14,6 +15,8 @@ def initialize(options = {})
           raise ArgumentError, "Missing :key"
         end
 
+        @fallback_key = options.fetch(:fallback_key, nil)
```
Comment (rubemz) [inline lib/crypt_keeper/provider/postgres_pgp.rb]: can you doc the fallback_key as an option?

Comment (fabiokr) [reply lib/crypt_keeper/provider/postgres_pgp.rb]: c98ca41

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 SPEC_ROOT           = Pathname.new File.expand_path File.dirname __FILE__
 AR_LOG              = SPEC_ROOT.join('debug.log').to_s
 ENCRYPTION_PASSWORD = "supermadsecretstring"
+FALLBACK_ENCRYPTION_PASSWORD = "fallbacksecretstring"
```
Comment (rubemz) [inline spec/spec_helper.rb]: can you align the assignment?
the above ones are aligned

Comment (fabiokr) [reply spec/spec_helper.rb]: Fixed

**Comment by fabiokr** [comment]
Closing this as we decided it is better to migration everything.

**Pull request closed without merging**